benchmark/kvbench/commands/nixlbench.py

```
@@ -228,13 +228,25 @@ def _configure_ucx(self, backend: str, source: str, destination: str):
+        if source in arg_to_seg_type:
+            self.op_type = "WRITE"
+            self.initiator_seg_type = arg_to_seg_type[source]
+        elif destination in arg_to_seg_type:
+            self.op_type = "READ"
+            self.initiator_seg_type = arg_to_seg_type[destination]
         else:
-            raise ValueError(f"Invalid source for OBJ: {source}")
+            raise ValueError(
+                f"Invalid source/destination for OBJ: source={source}, destination={destination}, "
+                f"one of them must be in {list(arg_to_seg_type.keys())}"
+            )
```

Ruff (0.15.10)

[warning] Avoid specifying long messages outside the exception class

(TRY003)